### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by brunolemos**

### CODE REVIEW

File: packages/theme-ui/README.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ export default props => (
 )
 ```
 
-The `theme` object follows the System UI [Theme Specification](/theme-spec),
+The `theme` object follows the System UI [Theme Specification](/packages/docs/src/pages/theme-spec.mdx),
```
Comment (lachlanjc) [inline packages/theme-ui/README.md:80]: This breaks the link, I’m pretty sure—the page is live here: https://theme-ui.com/theme-spec/ but this link doesn’t work: https://theme-ui.com/packages/docs/src/pages/theme-spec.mdx

Comment (brunolemos) [reply packages/theme-ui/README.md:80]: ah, I'm basing it on the github readme, not the website. so I guess it needs to be an absolute URL to work on both.

Comment (jxnblk) [reply packages/theme-ui/README.md:80]: Links like this are likely a result of copying and pasting links from the docs site source, but for links like this in the READMEs, they should probably be absolute URLs

**Comment by jxnblk** [comment]
Awesome, thanks!!

**Pull request merged**